sdk/identity/identity/package.json

```diff
@@ -43,7 +44,7 @@
     "test:node": "npm run clean && npm run build:test && npm run unit-test:node && npm run integration-test:node",
     "test": "npm run clean && npm run build:test && npm run unit-test && npm run integration-test",
     "unit-test:browser": "karma start",
-    "unit-test:node": "mocha test-dist/**/*.js --reporter ../../../common/tools/mocha-multi-reporter.js",
+    "unit-test:node": "mocha --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 180000 --full-trace test-dist/**/*.js",
```

Comment:
So, I both made this more standard with other clients... and added this extended timeout, because I haven't figured out yet how to bypass MSAL's own timeouts during playback. I tried with Sinon timers, but I didn't succeed. I'll make an issue.

Reply:
New issue: https://github.com/Azure/azure-sdk-for-js/issues/14267

Reply:
For unit tests, we shouldn't actually be exercising MSAL right? Since that's definitely not the unit under test, it's not even part of this package! 😉

Reply:
I'm using recorded tests. I am relying on the recorder and MSAL to avoid having to mock MSAL, since I want to test the real behavior, record what happened, then match against the recordings. This is what we do in many clients, and also what .Net is doing. Saves us time and gets us closer to reality. We can improve it over time.